fe/lib/components/Questionnaire/QuestionnaireForm/components/componentId.ts

```diff
@@ -1,5 +1,13 @@
 export const componentId = (prefix: string, ...rawValues: string[]) => {
-  const values = rawValues.map((value) => Buffer.from(value).toString('hex'));
+  const values = rawValues.map((str) => {
+    let value = '';
+
+    for (let i = 0; i < str.length; i++) {
+      value += str.charCodeAt(i).toString(36);
```

Comment:
`toString(16)` would be equivalent to the prior code, but figured I'd bump this while we're here